Figures/ObjectCanvas.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+//using System.Linq;
```

Comment:
remove these lines - keeping commented out lines in code w/o real need is a bad style. It distrusts the attention of the reviewer.